src/rapidfuzz/process_py.py

```diff
@@ -168,66 +174,66 @@ def extract_iter(
 
 @overload
 def extractOne(
-    query: Sequence[Hashable] | None,
-    choices: Iterable[Sequence[Hashable] | None],
+    query: _StringType | None,
+    choices: Mapping[_KeyType, _StringType | None],
```

Comment:
I think using the same `TypeVar` for both query and choices is probably problematic, since while it's true that they can be the same types, they can be of different types in an individual invocation. E.g. `process.extractOne(list(""), [""])` would be fine. I didn't test this yet though.